upgrade-instructions.md

```diff
@@ -26,6 +26,34 @@ Change list:
  - [Recommended upgrade to Cassandra 3.11.10](#recommended-upgrade-to-cassandra-3-1110)
  - [JMS mail queue no longer relies on java serialization](#jms-mail-queue-no-longer-relies-on-java-serialization)
 
+### Recommended upgrade to Elasticsearch 7.10.2
+
+Date: 11/03/2021
+
+JIRA: https://issues.apache.org/jira/browse/JAMES-2492
+
+Concerned products: Cassandra Guice server (with and without rabbitMQ)
+
+James is no longer tested against Elasticsearch 6.3.2 but instead against Elasticsearch 7.10.2 Users are recommended to upgrade to this
```

Comment:
`James is no longer tested against Elasticsearch 6.3.2`

Weknow it no longer is compatible, no?

Reply:
> Shall we rename packages and maven modules to remove the v7 suffix?

I thought that maybe if in the future a new version change is made, then it is good to already have the v7 packages, so it is easier to create separate modules for example elasticsearch-v8 (then just need to rename the package not create them from null)

What do you think?

Reply:
I tend to agree with you, this sounds like a good explanation.